.github/workflows/ci.yml

```diff
@@ -200,6 +200,10 @@ jobs:
         MS_TENANT_ID: ${{ secrets.MS_TENANT_ID }}
         MS_USER_EMAIL: ${{ secrets.MS_USER_EMAIL }}
         MS_USER_PNAME: ${{ secrets.MS_USER_PNAME }}
+        BOX_APP_CRED: ${{ secrets.BOX_APP_CRED }}
```

Comment:
not necessary as this was renamed and is defined [here](https://github.com/Unstructured-IO/unstructured/blob/b4fe40e4847d4517fc9ece58743222bf38ffe633/.github/workflows/ci.yml#L189)